js/modules/k6/grpc/client_test.go

```diff
@@ -1325,74 +1131,44 @@ func TestDebugStat(t *testing.T) {
 func TestClientInvokeHeadersDeprecated(t *testing.T) {
```

Comment:
It sounds like it could be the time to drop it completly.

Reply:
The test is still relevant for the existing functionality. However, I like the idea of dropping the `headers` property, but how about opening a separate PR  where we start erroring (right now, we just `fallthrough` to the `metadata`) and keep this for a cycle/too till we altogether remove it?

Reply:
I'm fine with it :+1: